stdlib/public/Concurrency/TaskPrivate.h

```diff
@@ -159,7 +159,7 @@ class TaskFutureWaitAsyncContext : public AsyncContext {
 } // end anonymous namespace
 
 /// The current state of a task's status records.
-class ActiveTaskStatus {
+class alignas(sizeof(void*) * 2) ActiveTaskStatus {
```

Comment:
Do you want to use `sizeof(uintptr_t)` to make the intent clearer?

Reply:
We use `void*` in other places, so I wanted to keep it consistent.